fe1-web/network/MessageApi.ts

```diff
@@ -289,16 +289,17 @@ export function terminateElection(
 }
 
 export function requestAddChirp(
-  text: string, parent_id: Hash = undefined,
+  text: string,
+  parentId?: Hash,
 ): Promise<void> {
   const timestamp = Timestamp.EpochNow();
   const currentLao: Lao = OpenedLaoStore.get();
 
   const message = new AddChirp({
     text: text,
-    parent_id: parent_id,
+    parent_id: parentId,
     timestamp: timestamp,
   });
 
-  return publish(channelFromIds(currentLao.id), message);
+  return publish(userSocialChannel(currentLao.id), message);
```

Comment:
does this channel already exist before the first chirp ? 
Is the server creating it automatically upon receiving a publish ? 🤔

Reply:
The channel is created when the user attends a roll call. For testing purposes, it is currently opened for the organizer as soon as he launches an LAO.